superset/translations/ru/LC_MESSAGES/messages.json

```
@@ -5619,7 +5624,7 @@
         "Вы можете создать новый график или использовать существующие из панели справа"
       ],
       "You can preview the list of dashboards in the chart settings dropdown.": [
-        ""
+        "Список дашбордов, в которые добавлен график, доступны в выпадающем списке возле кнопки \"\\Сохранить\"\\."
```

Could we refer to the chart settings dropdown itself rather than the button? Maybe something like this:
```suggestion
        "Вы можете посмотреть список дашбордов в выпадающем списке возле кнопки \"\\Сохранить\"\\."
```